src/main/java/ai/apptuit/metrics/jinsight/ConfigService.java

```diff
@@ -184,9 +184,44 @@ private static Properties loadProperties(File configFilePath) throws IOException
     try (InputStream inputStream = new BufferedInputStream(new FileInputStream(configFilePath))) {
       config.load(inputStream);
     }
+    // override with system properties, if present
+    config.putAll(loadSystemProperties());
     return config;
   }
 
+  private static Map<String, Object> loadSystemProperties() {
+    Map<String, Object> systemProperties = new HashMap<>();
+    String reporter = getProperty(REPORTER_PROPERTY_NAME);
```

Comment:
System prop name should be jinsight.reporter (even though the property name is just reporter)